voting/src/general_meeting.rs

```diff
@@ -186,6 +186,29 @@ impl TxOwner for Context {
     }
 }
 
+#[derive(Debug, Serialize, Deserialize)]
+pub enum Error {
+    InvalidMeeting,
```

Comment:
How about `MeetingNotFound` or `AbsentMeeting` or something?
The current name `InvalidMeeting` does not give the reader about what was wrong.